portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PagesList.testcase

```diff
@@ -36,6 +36,33 @@ definition {
 		ACUtils.tearDownAC();
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8343 | Test Summary: Custom Range time period filter in Page table"
+	@ignore = "true"
+	@priority = "5"
+	test CanTimeFilterUseCustomRangeInPageTable {
+
+		// TODO LRAC-8343 CanTimeFilterUseCustomRangeInPageTable
+
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8340 | Test Summary: 180 Days time period filter in Page table"
+	@ignore = "true"
+	@priority = "3"
+	test PeriodFilter180DaysPageTable {
+
+		// TODO LRAC-8340 ShowsListOfPages pending implementation
+
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8341 | Test Summary: Last Year time period filter in Page table"
+	@ignore = "true"
+	@priority = "3"
+	test PeriodFilterLastYearPageTable {
+
+		// TODO LRAC-8341 ShowsListOfPages pending implementation
+
```

Comment:
hey @LucasAcacio , please make sure the test on the TODO comment is the same of the test ;)